Metadata → Details

fiskus opened the pull request:
![Screenshot from 2022-06-13 13-35-09](https://user-images.githubusercontent.com/533229/173346855-8d9ba2c6-463c-4e0a-a320-3f2e38fa95f9.png)
![Screenshot from 2022-06-13 13-35-29](https://user-images.githubusercontent.com/533229/173346862-80bbe41c-4b6d-453e-a1b3-9309feb70c0f.png)

akarve commented:
This looks good but needs some wordsmithing:

* METADATA (same as before)
    * Message
    * User metadata
    * Workflow

akarve reviewed (commented):
This looks good but needs some wordsmithing:

* METADATA (same as before)
    * Commit message
    * User metadata
    * Workflow

Nice-to-have: on hover show JSON path to values displayed

nl0 commented on catalog/app/containers/Bucket/Meta.tsx:
```diff
@@ -0,0 +1,104 @@
+const HeadCell = ({ className, children, title }: HeadCellProps) => {
+  const classes = useHeadCellStyles()
+  return (
+    <M.TableCell className={cx(classes.root, className)} component="th" scope="row">
```
maybe vertically align this to the top? seems like it would look better that way

nl0 reviewed (approved):
im not 100% convinced with the "table" design, maybe it would look better without borders idk, but anyways it feels like a step into the right direction

fiskus commented:
* Removed vertical borders
* Vertically aligned table row titles
* Added quotes
* Renamed "Commit 
![Screenshot from 2022-06-23 12-29-03](https://user-images.githubusercontent.com/533229/175266662-b5a6a17d-39fb-43fe-a51f-abfa222e8262.png)
![Screenshot from 2022-06-23 12-29-16](https://user-images.githubusercontent.com/533229/175266666-0a099fe5-2790-471a-a051-c3487c9271cc.png)
message" to "Message"

nl0 commented:
> it would be nicer to not have quotes around message

> Added quotes

=) sorry for misleading comment, but seems like there were NO quotes at the moment of review, but i was commenting based on the screenshot which had quotes

also you could probably make the spacing between labels and values a little smaller

fiskus commented:
Yeah, missed "not" in "to not have quotes" :)
